packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -28,7 +30,16 @@ export const RadioGroup = compose<RadioGroupType>({
         return null;
       }
 
-      const { label, defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(radioGroup.props, final);
+      const { label, required, defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(radioGroup.props, final);
+
+      const labelComponent = (
+        <React.Fragment>
```

Comment:
I don't think this needs to be wrapped in a Fragment

Reply:
Removed!